.tx/config

```diff
@@ -1,3 +1,245 @@
+type = PO
+
+[qwat-doc.contributor-guide-index]
+source_file = i18n/pot/contributor-guide/index.pot
+source_lang = en
+trans.de = i18n/de/contributor-guide/index.po
+trans.fr = i18n/fr/contributor-guide/index.po
+trans.it = i18n/it/contributor-guide/index.po
+trans.ro = i18n/ro/contributor-guide/index.po
+type = PO
+
+[qwat-doc.demo-guide-index]
+source_file = i18n/pot/contributor-guide/index.pot
+source_lang = en
+trans.de = i18n/de/demo-guide/index.po
+trans.fr = i18n/fr/demo-guide/index.po
+trans.it = i18n/it/demo-guide/index.po
+trans.ro = i18n/ro/demo-guide/index.po
+type = PO
+
+[qwat-doc.developer-guide-index]
+source_file = i18n/pot/developer/guide-index.pot
+source_lang = en
+trans.de = i18n/de/developer-guide/index.po
+trans.fr = i18n/fr/developer-guide/index.po
+trans.it = i18n/it/developer-guide/index.po
+trans.ro = i18n/ro/developer-guide/index.po
+type = PO
+
+[qwat-doc.developer-guide-local-customizations]
+source_file = i18n/pot/developer-guide/local-customizations.pot
+source_lang = en
+trans.de = i18n/de/developer-guide/local-customizations.po
+trans.fr = i18n/fr/developer-guide/local-customizations.po
+trans.it = i18n/it/developer-guide/local-customizations.po
+trans.ro = i18n/ro/developer-guide/local-customizations.po
+type = PO
+
+[qwat-doc.developer-guide-qwat-dev]
+source_file = i18n/pot/developer-guide/qwat-dev.pot
+source_lang = en
+trans.de = i18n/de/developer-guide/qwat-dev.po
+trans.fr = i18n/fr/developer-guide/qwat-dev.po
+trans.it = i18n/it/developer-guide/qwat-dev.po
+trans.ro = i18n/ro/developer-guide/qwat-dev.po
+type = PO
+
+[qwat-doc.developer-guide-upgrade]
+source_file = i18n/pot/developer-guide/upgrade.pot
+source_lang = en
+trans.de = i18n/de/developer-guide/upgrade.po
+trans.fr = i18n/fr/developer-guide/upgrade.po
+trans.it = i18n/it/developer-guide/upgrade.po
+trans.ro = i18n/ro/developer-guide/upgrade.po
+type = PO
+
+[qwat-doc.faq-index]
+source_file = i18n/pot/faq/index.pot
+source_lang = en
+trans.de = i18n/de/faq/index.po
+trans.fr = i18n/fr/faq/index.po
+trans.it = i18n/it/faq/index.po
+trans.ro = i18n/ro/faq/index.po
+type = PO
+
+[qwat-doc.features-guide-index]
+source_file = i18n/pot/features/guide-index.pot
+source_lang = en
+trans.de = i18n/de/features-guide/index.po
+trans.fr = i18n/fr/features-guide/index.po
+trans.it = i18n/it/features-guide/index.po
+trans.ro = i18n/ro/features-guide/index.po
+type = PO
+
+[qwat-doc.glossary-index]
+source_file = i18n/pot/glossary/index.pot
+source_lang = en
+trans.de = i18n/de/glossary/index.po
+trans.fr = i18n/fr/glossary/index.po
+trans.it = i18n/it/glossary/index.po
+trans.ro = i18n/ro/glossary/index.po
+type = PO
+
+[qwat-doc.index]
+source_file = i18n/pot/index.pot
+source_lang = en
+trans.de = i18n/de/index.po
+trans.fr = i18n/fr/index.po
+trans.it = i18n/it/index.po
+trans.ro = i18n/ro/index.po
+type = PO
+
+[qwat-doc.installation-guide-database-initialization]
+source_file = i18n/pot/installation-guide/database-initialization.pot
```

Comment:
This is actually OK, but we could change the rst file name with an underscore if we want to be systematic with files namings

Reply:
That is a good idea, I'm just not sure if transifex will transfer all the existing translations or if that needs manual work.